index.js

```diff
@@ -136,13 +137,17 @@ module.exports = function(options) {
 
 		var agent =  options.https ? new https.Agent({maxSockets:workers}) : new http.Agent({maxSockets:workers}); // long poll should use its own agent
 		var req = request.defaults({agent:agent});
+		var pullRequestOptions = {};
+		if (options.waitTimeSeconds !== null) {
+			pullRequestOptions.WaitTimeSeconds = options.waitTimeSeconds
+		}
```

Comment:
options.waitTimeSeconds is never null because of the || 20 so this block is unnessary